README.md

```diff
@@ -5,16 +5,19 @@
 ![Coverage](coverage_badge.svg)
 
 * Python implementation of the [ENS Name Normalization Standard](https://github.com/adraffy/ensip-norm/blob/main/draft.md).
-  Thanks to [Adraffy](https://github.com/adraffy) for his leadership in coordinating the definition of this standard.
+  Thanks to [Adraffy](https://github.com/adraffy) for his leadership in coordinating the definition of this standard with the ENS community.
 * Passes **100%** of the [official validation tests](https://github.com/adraffy/ens-normalize.js/tree/main/validate) (validated automatically with pytest, see below).
 * Passes an [additional suite of further tests](/tools/updater/update-ens.js#L54) for compatibility with the official [Javascript reference implementation](https://github.com/adraffy/ens-normalize.js) and code testing coverage.
 * Based on [JavaScript implementation version 1.9.0](https://github.com/adraffy/ens-normalize.js/tree/4873fbe6393e970e186ab57860cc59cbbb1fa162).
 
 ## Glossary
 
-* disallowed name - name that cannot be converted into a valid normalized form using ENS Name Normalization Standard
-* valid name - name that is normalized or normalizable by the standard
-* fixable errors - `DisallowedLabelError`s for which fields `disallowed`, `index`, `suggested`, and `disallowed_sequence_info` are not None
+* normalized name - name that is already in normalized form according to the ENS Normalization Standard
+* normalizable name - name that is normalized or that can be converted into a normalized name using ens_normalize
+* disallowed name - name that is not normalized or normalizable
```

Comment:
if there is no definition of valid name then we should change "valid" from the rest of document